woocommerce/woocommerce-gutenberg-products-block
# Move `Spinner` to components package

## What

Moves the `Spinner` to the components package and updates imports used across the project to reflect the new location:

- `Button` (This is not imported from `@woocommerce/blocks-components` but it does use a relative import because of circular dependecies)
- `LoadingMask` 

## Why

As more developers integrate their extensions with WooCommerce Blocks, the case for reusing components grows. Rather than recreating components developers can access `wc.blocksComponents` and get the ones we need.

## Testing Instructions



_Please consider any edge cases this change may have, and also other areas of the product this may impact._

1. Ensure you have two or more shipping zones set up on your site and each zone contains at least one method.
2. Add an item to your cart and go to the Checkout block.
3. Change country and observe the shipping options list.
4. Ensure the spinner shows while the shipping methods update.
5. Fill in your details and place the order.
6. On the "Place order" button ensure a spinner shows while it processes.


## Screenshots or screencast



N/A

## WooCommerce Visibility


Required:


## Checklist

Required:

Conditional:

## Changelog


> Make the `Spinner` component available in the `@woocommerce/blocks-components` package.
